src/status_im2/contexts/wallet/events.cljs

```diff
@@ -57,9 +58,9 @@
                [:dispatch [:hide-bottom-sheet]]
                [:dispatch-later
                 [{:dispatch [:navigate-back]
-                  :ms       100}
```

Comment:
trying to fix toast bug, I can revert. either way we need to clean this up with correct transition eventually.